src/test/java/racingcar/winner/model/CarTest.java

```diff
@@ -15,8 +15,14 @@ public boolean canMove() {
             }
         });
 
-        Assertions.assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(1);
-        Assertions.assertThat(car.getMoveResult()).isNotEqualTo(movedCar.getMoveResult());
+        org.junit.jupiter.api.Assertions.assertAll(
+                () -> {
+                    Assertions.assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(1);
+                },
+                () -> {
+                    Assertions.assertThat(car.getMoveResult()).isNotEqualTo(movedCar.getMoveResult());
+                }
+        );
```

Comment:
각각 따로 static import 받으면 좋을 것 같아요. 중괄호도 생략 가능하겠네요!

```suggestion
        assertAll(
                () -> assertThat(movedCar.getMoveResult().getMoveCount()).isEqualTo(1),
                () -> assertThat(car.getMoveResult()).isNotEqualTo(movedCar.getMoveResult())
        );
```